lib/graphql-pundit.rb

```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: true
```

Comment:
I added the exception to `.rubocop.yml`, since the list there is overriding the config in `.rubocop_modified.yml`. Perhaps we should look these files through and sort them a bit (they are the same as in the backend repo).